lib/core/src/Cardano/Wallet/Types/Read/Tx.hs

```diff
@@ -69,11 +72,34 @@ but the type family `Api.ShelleyLedgerEra` performs the conversion.
 -- This transaction can come from any era, old (Byron) and new (Babbage).
 data Tx where
     Tx
-        :: forall era. Api.IsCardanoEra era
+        :: forall era
+        . (Show (TxEra era), Api.IsCardanoEra era)
         => Api.CardanoEra era
         -> TxEra era
         -> Tx
 
+deriving instance Show Tx
```

Comment:
👍🏻

Reply:
I would like to do something against the proliferation of type class constraints (e.g. `Show (TxEra era)`) — but I don't think I can do anything here at the moment. 👍 from me as well.